src/main/java/net/imglib2/roi/boundary/Boundary.java

```diff
@@ -0,0 +1,296 @@
+package net.imglib2.roi.boundary;
+
+import static net.imglib2.roi.boundary.Boundary.StructuringElement.FOUR_CONNECTED;
```

Comment:
I'll put the `StructuringElement` enum in a separate file in `imglib2-algorithm` so we can re-use it from now on...